src/renderer/theme.store.ts

```diff
@@ -32,42 +34,46 @@ export enum ThemeType {
 }
 
 export interface Theme {
-  id: ThemeId; // filename without .json-extension
   type: ThemeType;
```

Comment:
We can use literal types here `type: "dark" | "light"` and set them explicitly below. To avoid verbose things like
```
const darkTheme: Theme = {
  ...darkThemeBase,
  type: darkThemeBase.type as ThemeType,
};
```

Reply:
Okay sounds good